src/main/java/core/basesyntax/MyHashMap.java

```diff
@@ -1,19 +1,86 @@
 package core.basesyntax;
 
+import java.util.Objects;
+
 public class MyHashMap<K, V> implements MyMap<K, V> {
+    private static final int DEFAULT_CAPACITY = 16;
+    private static final float LOAD_FACTOR = 0.75F;
+    private static final int GROW_VALUE = 2;
+    private Node<K, V>[] table;
+    private int size;
+
+    class Node<K, V> {
```

Comment:
private class Node

[Private methods and classes should be after public ones in your class.](https://github.com/yreuta/jv-hash-map/blob/hw-1/checklist.md#private-methods-and-classes-should-be-after-public-ones-in-your-class)